### DISCUSSION TITLE
Incremental offsite backups

### DISCUSSION THREAD

**Discussion by spiroski** [minio/minio#15749] [Q&A] [answered]
Hello,

What would be the recommended approach to achieving an incremental backup to an offsite storage ( for example tapes ). 
Should we use mc mirror to mirror to a filesystem and run backups of that filesystem? 
Is it good idea to have a replica which we stop and then do a snapshot of the underlying filesystem and then resume the replica and then store the snapshot elswere?

What else is there and can someone confirm they have used such strategy in production?

P.S. offsite backup is a regulatory requirement for the software we provide, so built in functionalities of mini.io like versioning etc. is not enough. The data has to be stored elsewhere.

**Comment by abperiasamy** [comment, marked as answer]
Why not enable site replication, which also replicates versions, metadata and bucket level settings?. Copying buckets and objects using `mc mirror` or external backup tools (most of them support S3 API) will work, but they won't scale for petabytes of data. We are also about to release server-side distributed batch replication to make point-in-time copies.

**Reply by spiroski** [reply]
It is a business and regulatory requirement that I have to produce backups that can be ( and are ) stored offline, on something like tapes. Generally we need to store 30 days of daily backups, however there are situations that a customer asks to store a backup for longer themselves.
